From private repo

v-alje opened the pull request.

squillace commented:
Yes. We need to continue to do these kinds of clarifications -- often linking to either internal or external defs (wikipedia) really helps, too. :-)

coreysa commented:
Thanks, Ralph. I was actually contemplating linking to a definition of ACL in Wikipedia. We might want to continue to make these types of updates. Good idea.

v-vikum2 committed:
updated

ErikjeMS committed:
Update azure-stack-deploy.md

ErikjeMS committed:
Update azure-stack-deploy.md

CarlRabeler committed:
pools

Commit:
freshness updates

bandersmsft committed:
Log Analytics - wire data refresh

bandersmsft committed:
fixing links

bandersmsft committed:
fixing h1

BrucePerlerMS committed:
edit pass

bandersmsft committed:
fixed note

CarlRabeler committed:
more

CarlRabeler committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into 20170712-pools

dlepow committed:
Work in progress

BrucePerlerMS committed:
alter Psh fragment formatting - change from in-line to block

BrucePerlerMS committed:
tweek text

asd

Commit:
reperrorcodes

Commit:
reperrorcodestoc

BrucePerlerMS committed:
fix spacing around code block

dlepow committed:
work in progress

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into freshness

dlepow committed:
Acrolinx fixes

BrucePerlerMS committed:
tweak psh code fromat

barbkess committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into replicated

dlepow committed:
link fix

CarlRabeler committed:
more

dlepow committed:
edit

billmath committed:
freshness

BrucePerlerMS committed:
format tweak

CarlRabeler committed:
more

ronortloff committed:
Replicated table DMV

added link to Replicated table dmv

ronortloff committed:
Replicated table comments

added details on replicated tables.
removed identity limitation

CarlRabeler committed:
more

jeevansd committed:
Updati
ng SAML claim customization article

CarlRabeler committed:
more

Commit:
add steps for updating to V2

jeevansd committed:
Updating SAML claim customization article

Juliako committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into Analytics

Juliako committed:
updated a topic

Juliako committed:
updated a topic

CarlRabeler committed:
more

CarlRabeler committed:
more

CarlRabeler committed:
more

Commit:
Update analysis-services-data-providers.md

JasonWHowell committed:
Update connect-nodejs.md

JasonWHowell committed:
Update connect-nodejs.md

JasonWHowell committed:
Update connect-java.md

CarlRabeler committed:
typo

JasonWHowell committed:
Update connect-python.md

JasonWHowell committed:
Update connect-nodejs.md

barbkess committed:
remove cache wording

kgremban committed:
Fixed typos

nizarq committed:
Adding MonagentHost logs information.

Commit:
Merge pull request #17195 from JasonWHowell/patch-9

MySQL: Update connect-python.md

Juliako committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into Analytics

Juliako committed:
updated a topic

v-thepet committed:
fix redirect document id

Commit:
Merge pull request #17199 from nizarq/master

Adding MonagentHost logs information.

Commit:
Merge pull request #17182 from billmath/aadconnect

freshness

Commit:
Merge pull request #17185 from ronortloff/master

Replicated table DMV

Commit:
Merge pull request #17191 from JasonWHowell/patch-7

MySQL: Update connect-java.md

Lisaco88 committed:
Merge pull request #17190 from Minewiskan/patch-11

Update analysis-services-data-providers.md

Juliako committed:
Update media-services-rest-how-to-use.md

cjgronlund committed:
ds for beginners updates

cjgronlund committed:
ds for beginners updates

cjgronlund committed:
date

bandersmsft committed:
fixing note and typo

gbowerman committed:
Remove IPv6 LB info during private preview

To do: add it back during public preview

tynevi committed:
Update tools-and-setup.md

Commit:
Updating Known Issue for PS

markgalioto committed:
adding the redirect information to backup-ibiza-faq

JasonWHowell committed:
Update quickstart-create-server-database-portal.md

Lisaco88 committed:
Merge pull request #17204 from v-thepet/today

production bug: fix redirect document id

Lisaco88 committed:
Merge pull request #17161 from bandersmsft/master

Log Analytics - wire data refresh

v-thepet committed:
Merge pull request #17024 from JasonWHowell/mysqlnodejs

MySQL: Quickstart connect & query NodeJS

Juliako committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into Analytics

Juliako committed:
updated a topic

Commit:
Merge pull request #17206 from Juliako/patch-1

Update media-services-rest-how-to-use.md

v-thepet committed:
Merge pull request #17207 from cjgronlund/fresh-july

Freshness - DS for Beginners updates

Juliako committed:
Update media-services-portal-vod-get-started.md

Commit:
Merge pull request #16462 from DeniseMak/update-7-2

explanation of resolution and value field

v-thepet committed:
Merge pull request #17113 from Juliako/Analytics

updated analytics docs

Commit:
Merge pull request #17212 from Juliako/patch-2

Update media-services-portal-vod-get-started.md

v-thepet committed:
Merge pull request #17210 from markgalioto/Az7-12

adding the redirect information to backup-ibiza-faq

v-thepet committed:
Merge pull request #17208 from gbowerman/patch-6

Remove IPv6 LB info during private preview

billmath committed:
freshness2

Commit:
Merge pull request #17209 from HeathL17/hl-relnotes

Fix formatting

Commit:
Merge pull request #17215 from billmath/aadconnect

freshness2

billmath committed:
updating fresh

Commit:
Update iot-hub-adafruit-feather-m0-wifi-kit-arduino-get-started.md

Commit:
Update iot-hub-device-management-iothub-explorer.md

Commit:
Update iot-hub-intel-edison-kit-c-get-started.md

Commit:
Merge pull request #17217 from shizn/patch-17

Update iot-hub-intel-edison-kit-c-get-started.md

Commit:
Merge pull request #17218 from shizn/patch-15

Update iot-hub-adafruit-feather-m0-wifi-kit-arduino-get-started.md

Commit:
Merge pull request #17219 from shizn/patch-16

Update iot-hub-device-management-iothub-explorer.md

barbkess committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into replicated

dlepow committed:
Freshened scp article

dlepow committed:
typo

dlepow committed:
Acrolinx fixes

barbkess committed:
ron's updates

barbkess committed:
replicated tables in toc

barbkess committed:
add images

barbkess committed:
fix note

barbkess committed:
rearrange

gsacavdm committed:
Fixing redirect in create-vm-generalized

barbkess committed:
fix link

mahesh-unnikrishnan committed:
Update active-directory-ds-getting-started-create-group.md

anoobbacker committed:
Updating steps for service encryption rollover

Using powershell scripts steps added.

Commit:
Merge pull request #17226 from mahesh-unnikrishnan/master

Update active-directory-ds-getting-started-create-group.md

v-viinde committed:
Updated

v-viinde committed:
Updated

v-vikum2 committed:
updated

v-viinde committed:
USER STORY 1024855 SaaS App Tutorial:  EmpCenter Update

v-vikum2 committed:
updated

v-nagta committed:
Updated

anoobbacker committed:
Minor corrections

v-awku committed:
USER STORY 1029923 SaaS App Tutorial: SCC LifeCycle Tutorial

dominicbetts committed:
Add SAS to glossary

v-vikum2 committed:
updated

v-viinde committed:
Updated

Commit:
quickfix01

Commit:
Merge pull request #17237 from MarkusVi/quickfix01

quickfix01

Commit:
Merge pull request #17216 from billmath/aadconnect

updating fresh

Commit:
Merge pull request #17240 from prateek9us/master

add sql 2016 support

johnpapa committed:
fixing as per review

johnpapa committed:
fixing as per review

v-awku committed:
USER STORY 1029923 SaaS App Tutorial: SCC LifeCycle  Tutorial

billmath committed:
updating fresh3

Commit:
Merge pull request #17242 from billmath/aadconnect2

updating fresh3

billmath committed:
updating fresh4

Commit:
Merge pull request #17244 from billmath/aadconnect2

updating fresh4

billmath committed:
fresh5

Commit:
Merge pull request #17245 from billmath/aadconnect2

fresh5

billmath committed:
fresh6

Commit:
freshness

Lisaco88 committed:
Merge pull request #16694 from jlian/offer-quota

Added info about resetting quota increases during offer conversion

georgewallace committed:
application gateway : update ip addresses

BrucePerlerMS committed:
process feedback from

Commit:
Merge pull request #17246 from billmath/aadconnect2

fresh6

Commit:
revising

rjagiewich committed:
Merge pull request #17225 from anoobbacker/master

Updating steps for service encryption rollover

BrucePerlerMS committed:
tweak list format

Commit:
Merge pull request #17211 from JasonWHowell/patch-10

PostgreSQL: Update quickstart-create-server-database-portal.md

tdykstra committed:
simplify app service app entry

tdykstra committed:
typo

rjagiewich committed:
Merge pull request #17222 from gsacavdm/patch-3

Fixing redirect in create-vm-generalized

tdykstra committed:
fix link

rjagiewich committed:
Merge pull request #17165 from BrucePerlerMS/brp-work

Edit pass

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

kgremban committed:
Update application-proxy-teams.md

v-alje committed:
Merge pull request #17250 from MicrosoftDocs/FromPublicRepo

From public repo

Commit:
Merge pull request #17248 from georgewallace/patch-3

application gateway : update ip addresses

georgewallace committed:
application gateway : update ports

tynevi committed:
Update git-commands-for-master.md

Commit:
Merge pull request #17251 from georgewallace/patch-4

application gateway : update ports

rjagiewich committed:
Merge pull request #17155 from CarlRabeler/20170712-pools

pools

tynevi committed:
Update release-branches.md

tdykstra committed:
Merge pull request #1 from dominicbetts/glossary

Add SAS to glossary

jeevansd committed:
Updating the SAML claims article with new portal screens

anosov1960 committed:
Update sql-database-long-term-retention.md

CarlRabeler committed:
adaptive

ErikjeMS committed:
Update azure-stack-deploy.md

ErikjeMS committed:
updates for GA

Commit:
Merge pull request #17254 from anosov1960/patch-1

Update sql-database-long-term-retention.md

v-shils committed:
Merge pull request #17247 from Blackmist/freshness

Freshness

jeevansd committed:
Updating the SAML claims article with new portal screens

ErikjeMS committed:
Add files via upload

v-thepet committed:
Merge pull request #15934 from tdykstra/glossary

update Azure glossary

ErikjeMS committed:
Update azure-stack-marketplace-azure-items.md

v-thepet committed:
Merge pull request #13559 from v-viinde/googleappsv2

USER STORY 956545 SaaS App Tutorial: Google Apps

CarlRabeler committed:
Update sql-database-compatibility-level-query-performance-130.md

v-thepet committed:
Merge pull request #16568 from barbkess/replicated

Replicated

v-shils committed:
Merge pull request #17120 from jeevansd/SAMLClaimsCustomization

Updating SAML claim customization article

Commit:
Merge pull request #17255 from CarlRabeler/master

adaptive

v-thepet committed:
Merge pull request #16638 from kgremban/jul5_appproxyteams

New article on App Proxy + Teams

v-thepet committed:
Merge pull request #16723 from johnpapa/patch-1

review changes  to be clearer

barbkess committed:
fixes

tdykstra committed:
app service and sas

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into freshness

CarlRabeler committed:
Update sql-database-compatibility-level-query-performance-130.md

dlepow committed:
removed stale bookmark

ErikjeMS committed:
Update azure-stack-create-plan.md

ErikjeMS committed:
Add files via upload

tynevi committed:
Update contributor-guide-index.md

Commit:
Merge pull request #17261 from tdykstra/patch-1

app service and sas

ErikjeMS committed:
Add files via upload

KumudD committed:
content update

barbkess committed:
acrolinx

KumudD committed:
content update

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into freshness

dlepow committed:
typos

curtand committed:
freshness

v-thepet committed:
Merge pull request #16897 from MarkusVi/reperrorcodes

reperrorcodes

ErikjeMS committed:
Update azure-stack-create-plan.md

ErikjeMS committed:
updates for GA

barbkess committed:
clarity changes

Commit:
change to code path for 9.0 and 9.1

adegeo committed:
Minor format changes; fixed link

v-thepet committed:
Merge pull request #14405 from v-vikum2/atlassian

USER STORY 966476 SaaS App Tutorial: Atlassian Cloud Tutorial update

Commit:
Update backup-azure-arm-vms-prepare.md

v-shils committed:
Merge pull request #17135 from v-viinde/bime

USER STORY 1033870 SaaS App Tutorial: Bime Tutorial Update

Commit:
Merge pull request #17262 from barbkess/replicated

fixes

Commit:
Merge pull request #17267 from bradsev/path-fix-hadoop-r-server-edits

change to code path for 9.0 and 9.1

Commit:
Update storage-managed-disks-overview.md

Updated with Azure Backup non-support for Data disks >1TB.

Commit:
Merge pull request #17268 from trinadhk/patch-25

Update backup-azure-arm-vms-prepare.md

v-thepet committed:
Merge pull request #17127 from v-vikum2/thousandeyes

USER STORY 1032734 SaaS App Tutorial: ThousandEyes Tutorial Update

Commit:
Merge pull request #17269 from Thraka/vmss-net

Minor format changes; fixed link

Commit:
Merge pull request #17270 from trinadhk/patch-26

Update storage-managed-disks-overview.md

Commit:
Merge pull request #17264 from curtand/fresh0712

freshness

Commit:
Merge pull request #17265 from KumudD/TMUpdate0713

Content Update Traffic Manager Routing Methods

tynevi committed:
Update README.md

tynevi committed:
Update README.md

tynevi committed:
Update README.md

tynevi committed:
Update README.md

tynevi committed:
Update README.md

iainfoulds committed:
Adding explicit association between health probe and load balancer rule

Ja-Dunn committed:
Merge pull request #17107 from ErikjeMS/master

Turn off telemetry

nizarq committed:
Update sample configs:
Added updated placeholders for storage account information. Some customers were getting confused, thinking storage account can be empty if event hub is provided.

tynevi committed:
Update README.md

tynevi committed:
Update README.md

Lisaco88 committed:
Merge pull request #17075 from dlepow/freshness

VM article freshness

Commit:
Merge pull request #17277 from iainfoulds/vmwinmvcloadbalancer

VM Windows load balancer tutorial

tynevi committed:
fixing

Ja-Dunn committed:
Merge pull request #17258 from CarlRabeler/patch-299

Update sql-database-compatibility-level-query-performance-130.md

tynevi committed:
mcrez

Commit:
Merge pull request #17279 from nizarq/master

Update sample configs:

tynevi committed:
Merge pull request #17281 from tysonn/master

cg updates

Lisaco88 committed:
Merge pull request #17170 from MarkusVi/reperrorcodestoc

reperrorcodestoc

Commit:
Merge pull request #17273 from kfarmer-msft/azure-storage-emulator-5_2

Update release notes for Azure Storage Emulator v5.2

Lisaco88 committed:
Merge pull request #16179 from shruja/kantegajira

USER STORY 1027485 SaaS App Tutorial: Kantega SSO for JIRA

tynevi committed:
Update create-tables-markdown.md

tynevi committed:
Update custom-markdown-extensions.md

tynevi committed:
Update syntax-highlighting-markdown.md

tynevi committed:
Update style-and-voice.md

Lisaco88 committed:
Merge pull request #16622 from cherylmc/v2v

add peering link

Ja-Dunn committed:
Merge pull request #17229 from v-viinde/empcenter

USER STORY 1024855 SaaS App Tutorial:  EmpCenter Update

Ja-Dunn committed:
Merge pull request #17138 from v-vikum2/sapcloudforcustomer

USER STORY 966530 SaaS App Tutorial: SAP Cloud for Customer

Ja-Dunn committed:
Merge pull request #17137 from v-nagta/10000ftplansupdate

USER STORY 1035633 SaaS App Tutorial: 10,000ft Plans Update

Lisaco88 committed:
Merge pull request #16267 from shruja/kantegabamboo

USER STORY 1027540 SaaS App Tutorial: Kantega SSO for Bamboo

Lisaco88 committed:
Merge pull request #16377 from shruja/kantegafisheye

USER STORY 1029090: SaaS App Tutorial: Kantega SSO for FishEye/Crucible

Ja-Dunn committed:
Merge pull request #17128 from v-viinde/talentlms

USER STORY 966525 SaaS App Tutorial: TalentLMS Tutorial Update

Juliako committed:
Update media-services-dotnet-generate-thumbnail-with-mes.md

Lisaco88 committed:
Merge pull request #16476 from v-nagta/mypolicies

USER STORY 1030082 SaaS App Tutorial: myPolicies

Commit:
Merge pull request #17289 from Juliako/patch-1

Update media-services-dotnet-generate-thumbnail-with-mes.md

Juliako committed:
Update media-services-dotnet-how-to-use.md

Commit:
Merge pull request #17291 from Juliako/patch-1

Update media-services-dotnet-how-to-use.md

sankethka committed:
migration of empty mds with redirect tags to master redirection file

tynevi committed:
Merge pull request #17297 from MicrosoftDocs/test-redirects-final

migration of empty mds with redirect tags to master redirection file

Lisaco88 committed:
Merge pull request #17026 from shruja/workplacebyfacebookv2

USER STORY 956543 SaaS App Tutorial: Workplace by Facebook

Lisaco88 committed:
Merge pull request #16940 from v-vikum2/tidemark

USER STORY 1032735 SaaS App Tutorial: Tidemark Tutorial Update

Lisaco88 committed:
Merge pull request #16827 from v-awku/scclifecycle

USER STORY 1029923 SaaS App Tutorial: SCC LifeCycle Tutorial
